packages/demo/src/components/index.tsx

```diff
@@ -7,24 +7,25 @@ import ComponentPage from './ComponentPage';
 import {IComponent} from './ComponentsInterface';
 import SideMenu from './Menu';
 
+const componentFiles = require.context('./examples', true, /Examples?\.js?$/i, 'lazy');
+
+const load = async (path: string, ctx: any) => {
+    const component = await ctx(path);
+    const name = _.keys(component)[0].replace(/Examples?/i, '');
+    const componentPrototype = _.values(component)[0];
+    const c: IComponent = {
+        name,
+        path,
+        component: componentPrototype,
+    };
+    return c;
+};
+
+const loadAll = () => Promise.all(componentFiles.keys().map((path) => load(path, componentFiles)));
+

```

Comment:
Unrelated to the fix, just moving them out of the render function since they don't need to be.